packages/terra-functional-testing/lib/services/wdio-docker-service.js

```diff
@@ -0,0 +1,164 @@
+/* eslint-disable class-methods-use-this */
+const path = require('path');
+const util = require('util');
+const childProcess = require('child_process');
+const Logger = require('../logger/logger');
+
+const exec = util.promisify(childProcess.exec);
+const logger = new Logger({ prefix: 'wdio-docker-service' });
+
+class DockerService {
```

Comment:
Would this be better named, `LocalSeleniumService`? That seems to matter more than docker in this instance.

Reply:
How do you feel about meeting in the middle?

`SeleniumDockerService`

Reply:
I'll take it. `SeleniumDockerService` it is.

Reply:
Renamed to SeleniumDockerService:

https://github.com/cerner/terra-toolkit/pull/412/commits/98af84b2dcc2b282fe4b3bf350ac61487431ac9d